app/src/main/java/com/togetherjava/tjplays/listeners/commands/Game2048Command.java

```diff
@@ -30,7 +31,7 @@ public Game2048Command() {
     @Override
     public void onSlashCommandInteraction(SlashCommandInteractionEvent event) {
         if (!event.getName().equals(COMMAND_NAME)) return;
-
+        initialUser = event.getUser().getId();
```

Comment:
u have to send this in button id, and validate it in button interaction method

Reply:
Can you help me understand why I need to pass this to button ID? I tested this with another user on a server and they were unable to interact with the game I started while I could continue playing? 

Then they were able to start their own game and I could not interact with that one.

Reply:
This userId string is also validated in the button interaction event

Reply:
> Can you help me understand why I need to pass this to button ID? I tested this with another user on a server and they were unable to interact with the game I started while I could continue playing.

well thats because u write it to the entire command, so say if both of u start a game and each of u try to interact with others game only one will have desired result(not being able to interact), ur current solution will only work if theres only one game instance at a time which is totally not the case

> Then they were able to start their own game and I could not interact with that one. It seems like the functionality is working as intended.

now ask them to try interacting with urs again

Reply:
Ohhhhhh thank you :)
Okay so each user should be able to have a separate instance for their game, and it will be unique to them. 
So the suggested change is to add the "initialUser" string in the different buttons like this?

        Button resetButton = Button.success(COMMAND_NAME + initialUser + " reset", Emoji.fromUnicode("🔃"));

Then we do the contains check?

Reply:
> So the suggested change is to add the "initialUser" string in the different buttons like this?

yes, but add a space between `COMMAND_NAME` and `initialUser` also rename it to `playerId` that sounds more specific and clear

> Then we do the contains check?

yes

Reply:
Okay I am testing it now and I see what you mean. I need to remove the initialUser save from the Slash Command. I am trying to find out how to create unique instances of the game for others though...

Reply:
just pass it in button id and check in button interaction method thats all u have to do, everything else is managed by jda

Reply:
Do you mind joining this server to chat? I can share my screen
https://discord.gg/hShwmemt

Reply:
Fixed! Thank you!